src/gluonnlp/embedding/token_embedding.py

```diff
@@ -999,6 +999,8 @@ class Word2Vec(TokenEmbedding):
     embedding_root : str, default '$MXNET_HOME/embedding'
         The root directory for storing embedding-related files.
         MXNET_HOME defaults to '~/.mxnet'.
+    preload : bool, default True
+        Decide to load default model or not.
```

Comment:
if the goal is to construct an empty Word2Vec class object, how about supporting `source=None`?

Reply:
right, let me update it.